Makefile

```diff
@@ -0,0 +1,24 @@
+release:
+	rm -rf dist/
+	python setup.py sdist bdist_wheel
+	twine upload dist/*
+
+isort:
+	isort --remove-import . --apply --use-parentheses --trailing-comma
```

Comment:
these options are not necessary because the settings in `setup.cfg`?

Reply:
Actually. I have a new way of implementing this with pre-commit hooks, and GH actions checks for those who don't have pre-commits. Let me update this PR and, if you like that, we can implement it on `folium` too.

Reply:
Alright, sounds interesting!